py3status/modules/spotify.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+"""
+This module displays the current "artist - title" playing in Spotify.
+
+Last modified: 2015-03-26
+Author: Pierre Guilbert <pierre@1000mercis.com>
+License: GNU GPL http://www.gnu.org/licenses/gpl.html
+"""
+
+from time import time
+import dbus, dbus.proxies
+
+class Py3status:
+
+    # available configuration parameters
+    cache_timeout = 0
```

Comment:
having a format placeholder to let users configure a **format** parameter would be great ! can you do this plz ?

see the https://github.com/ultrabug/py3status/blob/master/py3status/modules/net_rate.py module for an example of format and placeholder usage